en/tutorials-and-examples/cms/authentication.rst

```diff
@@ -1,356 +1,283 @@
 CMS Tutorial - Authentication
 #############################
 
-Now that our CMS has users, we should enable them to login, and apply some basic
-access control to the article creation & editing experiences.
+Now that our CMS has users, we can enable them to login using the
+`cakephp/authentication <https://book.cakephp.org/authentication/2>`__
+plugin. We are going to ensure passwords are stored securely in
+our database, we are going to provide a working login page, and apply some
+basic access control to the article creation & editing experiences.
+
+Installing Authentication Plugin
+================================
+
+Use composer to install the Authentication Plugin::
+
+.. code-block:: bash
+
+    composer require cakephp/authentication:^2.0
+
 
 Adding Password Hashing
 -----------------------
 
-If you were to create/update a user at this point in time, you might notice that
+We will assume you have created the ``Controller``, ``Table``, ``Entity`` and
+templates for the ``users`` table in your database. You can do this manually
+like you did before for the ArticlesController, or you can use the bake shell
+to generate the classes for you using ``bin/cake bake all users``.
```

Comment:
Let's make this a proper code block to follow convention and not hide commands as this is the recommended practice.